tests/data/runFull/config.json

```diff
@@ -32,7 +32,7 @@
             "dbName": "name",
             "type": "nvarchar",
             "size": "255",
-            "nullable": null,
+            "nullable": false,
```

Comment:
Najprv som si myslel, ze je problem, ak je `nullable` `null` a nie `false`. Bol to omyl, ... no kedze sa to nikde netestuje, tak tu nechavam tuto hodnotu, aby to bolo aj niekde v testoch. `null` a `false` maju v tomto pripade rovnaky vyznam, UI generuje `false`.